helm/idol-stack/Chart.yaml

```
@@ -17,17 +17,17 @@ description: An IDOL setup
 dependencies:
 - name: idol-licenseserver
   condition: idol-licenseserver.enabled
-  version: "0.2.0"
+  version: "0.4.0"
   repository: file://../idol-licenseserver
```

Should make all of  these target published charts
```suggestion
  repository: https://raw.githubusercontent.com/opentext-idol/idol-containers-toolkit/main/helm
```